tools/prune-old-builds.py

```diff
@@ -0,0 +1,50 @@
+import os
+import shutil
+import subprocess
+from pathlib import Path
+from datetime import datetime
+
+FOURTEEN_DAYS = 14
+
+
+def how_old(date: str) -> int:
+    """Return a integer that represents how many days has passed given a date"""
+    return (datetime.now() - datetime.strptime(date, '%Y-%m-%d')).days
+
+
+if __name__ == '__main__':
+    builds_path = Path('builds/')
```

Comment:
The script should accept this path as an argument. Same with the Git repo checkout path.